saleor/webhook/event_types.py

```diff
@@ -18,6 +18,10 @@ class WebhookEventType:
     ORDER_CANCELLED = "order_cancelled"
     ORDER_FULFILLED = "order_fulfilled"
 
+    DRAFT_ORDER_CREATED = "draft_order_created"
```

Comment:
You didn't add a label and permission for this webhook
See below 👇  a `DISPLAY_LABELS`, `CHOICES` and `PERMISSIONS`

Reply:
@jakubkuc We need to update this as well.

Reply:
fixed it